### PR TITLE
Taxon styling tweaks

### PR DESCRIPTION
**Pull request by alecgibson**
These styling changes bring `collections` better in-line with the current Education Navigation prototype.

Some paragraph styling has been adopted from `govuk_elements`, which is used in the prototype but not (currently) in `collections`.

### Before

accordion-expanded-styling-before

### After

accordion-expanded-styling-after

### Before

accordion-no-js-mobile-before

### After

accordion-no-js-mobile-after

### Before

content-spacing-before

### After

content-spacing-after

### Before

content-spacing-mobile-before

### After

content-spacing-mobile-after

### Before

taxon-description-spacing-before

### After

taxon-description-spacing-after

Trello: https://trello.com/c/kSRKSoI2/326-review-styles-on-tablet-and-mobile-viewports-on-the-new-navigation-pages

### CODE REVIEW

**Comment by alextea** [comment]
@alecgibson Can you add screenshots of before and after?

**Comment by alecgibson** [comment]
@alextea done

File: app/assets/stylesheets/views/_taxons.scss [hunk 1]
```diff
@@ -23,13 +21,20 @@
...
   p {
     @include core-19;
-    margin-top: 2px;
-    margin-bottom: 6px;
+    margin-top: #{(5 / 16)}em;
+    margin-bottom: #{(20 / 16)}em;
+
+    @include media(tablet) {
+      margin-top: #{(5 / 19)}em;
+      margin-bottom: #{(20 / 19)}em;
```
Comment (suzannehamilton) [inline app/assets/stylesheets/views/_taxons.scss]: What's the significance of these values, and could it be made clearer with variables or comments? I might be missing some context, but it's not obvious to me why 20/19 em would be a useful measurement.

Comment (alextea) [inline app/assets/stylesheets/views/_taxons.scss]: @suzannehamilton the type size is 19px, so 20/19em will give you a 20px margin (and we use multiples of 5px for vertical rhythm). I'm not sure why we are using ems though.

Comment (alextea) [inline app/assets/stylesheets/views/_taxons.scss]: @alecgibson it's been suggested by other frontend devs on slack that we use the `$gutter` variables for this.

`margin-top: $gutter-two-thirds;`
`margin-bottom: $gutter-one-third / 2;`

Comment (alecgibson) [inline app/assets/stylesheets/views/_taxons.scss]: @alextea is it okay to use these values for all breakpoints?

**Pull request merged**